### PR TITLE
Refactor calculate-rem

### PR DESCRIPTION
**Pull request by codemonkey800**
## Description

Follow up to https://github.com/chanzuckerberg/napari-hub/pull/69#discussion_r647656269

### CODE REVIEW

File: client/src/utils/styles.scss [hunk 1]
```diff
@@ -16,8 +16,11 @@ $min-content: strip(theme('screens.screen-300'));
...
- * Function for calculating the rem value for another value.
+ * Function for calculating the rem value for another value using a base font
+ * size. The default base font size is 16px since that's what's used for most
+ * browsers:
+ * https://stackoverflow.com/a/29512572
  */
-@function calculate-rem($value) {
-  @return $value / 16px * 1rem;
+@function calculate-rem($value, $base-font-size: 16px) {
+  @return ($value / $base-font-size) * 1rem;
```
Comment (codemonkey800) [inline client/src/utils/styles.scss:25]: I couldn't do `$value / $base-font-size) rem` unfortunately, chrome complains about the value for some reason

Comment (kne42) [reply client/src/utils/styles.scss:25]: huh weird, what happens if you do
```suggestion
  @return #{($value / $base-font-size)} rem;
```

Comment (codemonkey800) [reply client/src/utils/styles.scss:25]: still doesn't work unfortunately 😢 
image

Comment (kne42) [reply client/src/utils/styles.scss:25]: ah, it must be the space or something. doesn't really matter anyways it's fine as is

**Comment by codemonkey800** [comment]
I'll go ahead and merge this since it's so small and previous comments seem to implicitly approve this :^)

**Pull request merged**